spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -190,6 +189,40 @@ public void delete(@PathVariable("name") String name) {
 	}
 
 	/**
+	 * Return a list of related stream definition resources based on the given stream name.
+	 * Related streams include the main stream and the tap stream(s) on the main stream.
+	 *
+	 * @param name the name of an existing stream definition (required)
+	 */
+	@RequestMapping(value = "/{name}/related", method = RequestMethod.GET)
```

Comment:
@aclement @BoykoAlex It might be helpful to have the same REST endpoint `/streams/definitions/{name}` to serve the related streams (with some request attribute set) response as well. But that will make the response resulting in single/plural response based on the attribute. Hence, having this explicit REST endpoint might be better. What do you think?